docs/editor/codebasics.md

```diff
@@ -206,6 +206,40 @@ Example:
 
 The modifiers can also be stacked - for example, `\u\u\u$1` will uppercase the first three characters of the group, or `\l\U$1` will lowercase the first character, and uppercase the rest. The capture group is referenced by `$n` in the replacement string, where `n` is the order of the capture group.
 
+## Search Editor
+Search Editors let you view workspace search results in a full-sized editor, complete with syntax highlighting and optional lines of surrounding context.
+
+Below is a search for the word 'SearchEditor' with two lines of text before and after the match for context:
+
+
+![search editor overview](images/codebasics/search-editor-overview.png)
+
+The Open Search Editor command opens an existing search editor if one exists, or to otherwise create a new one. The pre-existing command Open new Search Editor has been renamed to New Search Editor, and will always create a new Search Editor.
```

Comment:
Since this is for the main docs, you don't need to mention the previous command nam.

Reply:
So I'll get rid of the sentence: "The pre-existing command Open new Search Editor has been renamed to New Search Editor, and will always create a new Search Editor."

Reply:
I think you still want to mention the **New Search Editor** command and that it will always create a new Search Editor.

Reply:
Ah I get it, so Ill just write: The command **New Search Editor** will always create a new Search Editor.